test/test_native_balances.py

```diff
@@ -0,0 +1,50 @@
+import time
+import unittest
+
+import base
+from helpers.field_enums import NativeBalanceFields
+
+
+class TestNativeBalances(base.Base):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.clean_db({"native_balances"})
+
+        tx = cls.ledger_client.send_tokens(cls.delegator_wallet.address(), 10000, "atestfet", cls.validator_wallet)
+        tx.wait_to_complete()
+        if not tx.response.is_successful():
+            raise Exception(f"first set-up tx failed")
```

Comment:
At the time, I mistakenly thought that the assertions were instance methods but they should be class methods:

```suggestion
        cls.assertTrue(tx.response.is_successful(), f"first set-up tx failed")
```